cmd/copy.go

```diff
@@ -1449,7 +1449,7 @@ func (cca *CookedCopyCmdArgs) getSrcCredential(ctx context.Context, jpo *common.
 			// get token from env var or cache
 			if tokenInfo, err := uotm.GetTokenInfo(ctx); err != nil {
 				return srcCredInfo, err
-			} else if _, err := tokenInfo.GetTokenCredential(); err != nil {
+			} else if _, err := tokenInfo.GetTokenCredential(true); err != nil {
```

Comment:
Same here, why are we hardcoding the persistence behavior?

Reply:
Done in the latest commit.